ateam_kenobi/src/in_play_eval.hpp

```diff
@@ -0,0 +1,131 @@
+// Copyright 2021 A Team
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+#ifndef IN_PLAY_EVAL_HPP_
+#define IN_PLAY_EVAL_HPP_
+
+#include <math.h>
+#include <CGAL/squared_distance_2.h>
+#include <Eigen/Dense>
+#include <chrono>
+#include <optional>
+#include "types/world.hpp"
+#include <ateam_geometry/types.hpp>
+#include <ateam_common/game_controller_listener.hpp>
+
+
+/**
+ *  Small class to be "ticked" with the world output_state and keep track of if we enter a kickoff and then when we exit (ball has moved 0.05 after normal_start is issued, 10 seconds pass, force start is issued)
```

Comment:
It's unclear from the rules and I don't quite remember from comp... Is a force start automatically sent 10 seconds after kickoff if the ball doesn't move 0.05m? We don't seem to keep track of time this class, so I'm trying to figure out whether we actually need to.

Reply:
I don't think the rules specify that a force start happens automatically, but I think in practice a force start is called. Probably don't need to worry about this for merging.